.github/workflows/main.yml

```diff
@@ -21,6 +21,14 @@ jobs:
            env:
               FONT_AWESOME_NPM_TOKEN: ${{ secrets.FONT_AWESOME_NPM_TOKEN }}
            run: pnpm install
+         - name: Cache Cypress
+           id: cache-cypress
+           uses: actions/cache@v1
+           with:
+             path: ~/.cache/Cypress
+             key: cypress-cache-v2-${{ runner.os }}-${{ hashFiles('pnpm-lock.yaml') }}
```

Comment:
If this is an isolated machine that will run on `ubuntu` does `runner.os` need to be included for the cache key?

Reply:
I copied this little chunk from the cypress github action docs and it seemed reasonable. I'm inclined to leave it because it's safer (cache's won't be shared if we change OSes).